src/components/EventsList.tsx

```diff
@@ -20,9 +20,6 @@ const getDateText = (event: Event) => {
 
 const EventsList = ({ events, count }: EventsListProps) => {
   events = events.slice(0, count);
```

Comment:
I am actually not sure why we need this `slice` here haha. could you remove it?

Reply:
Yeah honestly I removed that too before since it seemed pretty useless but then I thought maybe there was a plans for it in the future undid my change lol. np

Reply:
Actually it seems like we use it to showcase only the next two upcoming events in the index page so I added it back